models/DataObject/Localizedfield/Dao.php

```diff
@@ -135,9 +135,12 @@ public function save($params = [])
          */
         DataObject\Concrete\Dao\InheritanceHelper::setUseRuntimeCache(true);
         foreach ($validLanguages as $language) {
-            if ((!isset($params['newParent']) || !$params['newParent']) && isset($params['isUpdate']) && $params['isUpdate'] && !$this->model->isLanguageDirty(
-                    $language
-                )) {
+            if ((!isset($params['newParent']) || !$params['newParent'])
+                && isset($params['isUpdate'])
+                && $params['isUpdate']
+                && !$this->model->isLanguageDirty($language)
+                 && !$forceUpdate
+                ) {
```

Comment:
`!isset($params[...]) || !$params[...]` can be simplified to `empty($params[...])` 
and
`isset($params[...]) && $params[...]` can be simplified to `!empty($params[...])` 

```suggestion
            if (empty($params['newParent'])
                && !empty($params['isUpdate'])
                && !$this->model->isLanguageDirty($language)
                && !$forceUpdate
            ) {
```